apigw-websocket-api-lambda-authorizer/apigw-websocket-api-lambda-authorizer.json

```diff
@@ -0,0 +1,100 @@
+{
+    "title": "WebSocket API Cognito Authentication using Lambda Authorizer",
```

Comment:
Please change to "Amazon Cognito authentication with Amazon API Gateway Websockets"